stestr/repository/file.py

```diff
@@ -19,12 +19,22 @@
 import sys
 import tempfile
 
-from six.moves import dbm_gnu as dbm
+import six
 from subunit import TestProtocolClient
 import subunit.v2
 import testtools
 from testtools.compat import _b
 
+if sys.platform != 'win32':
+    from six.moves import dbm_gnu as dbm
+else:
+    # TODO(claudiub): replace import with six.moves.dbm once the dbm move is
+    # defined: https://github.com/benjaminp/six/issues/179
+    if six.PY2:
+        import anydbm as dbm
```

Comment:
So my only concern here is that this uses dbhash by default on python 2 and it doesn't exist at all on python 3. The reason I switched to gdbm is that it existed on both versions of python (and a foolishly assumed all OSes) I'd like to avoid the bug where you have to run with python 3 first is there another dbm type that exists on windows?

Medium term I want to remove the dbm usage because there isn't any reason it has to be dbm. (and long term make the sql repo type the default)